GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -35,7 +30,7 @@
  * Descendant classes all have native (JNI) implementations; this base class
  * manages the native lifecycles.
  */
-public abstract class GVRHybridObject implements Closeable {
+public abstract class GVRHybridObject {
```

Comment:
The "Closeable" has been a misuse of the interface, which has more bulit-in meaning than the interface itself. It's good to remove it or create our own version of that.